src/control/cam_esmf_mod.F90

```diff
@@ -0,0 +1,167 @@
+module cam_esmf_mod
+
+  use shr_kind_mod      , only : r8=>shr_kind_r8, i8=>shr_kind_i8, cl=>shr_kind_cl, cs=>shr_kind_cs
+  use ESMF              , only : ESMF_Mesh, ESMF_Clock
+  use ESMF              , only : ESMF_VM, ESMF_VMAllreduce, ESMF_VMGetCurrent
+  use ESMF              , only : ESMF_SUCCESS, ESMF_REDUCE_SUM
+  use shr_sys_mod       , only : shr_sys_abort
+  use cam_abortutils    , only : endrun
```

Comment:
I don't think this is called (but `shr_sys_abort` is):
```suggestion
```

Reply:
Now calling endrun and not shr_sys_abort.